<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellPane</class>
 <widget class="QWidget" name="CellPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonExpand">
       <property name="text">
        <string>Expand all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCollapse">
       <property name="text">
        <string>Collapse all</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>148</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonRestoreDefaultCheck">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCheckAll">
       <property name="text">
        <string>All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCheckNone">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>37</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <column>
      <property name="text">
       <string notr="true">Cell</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Visible</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <zorder>treeWidget</zorder>
  <zorder>pushButtonRestoreDefaultCheck</zorder>
  <zorder>pushButtonCheckAll</zorder>
  <zorder>pushButtonCheckNone</zorder>
  <zorder>pushButtonExpand</zorder>
  <zorder>pushButtonCollapse</zorder>
  <zorder>treeWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
